src/bot/commands/userStats.ts

```diff
@@ -1,6 +1,6 @@
 import Command from '../lib/command';
 import { ApplicationCommandType, UserContextMenuCommandInteraction } from 'discord.js';
-import Stats from '../modules/stats';
+import { Stats } from '../modules/stats';
```

Comment:
```suggestion
import { Stats } from '../modules';
```
ce serait bien de faire cette petite modif dans tout les fichiers qui demande ce fichier